service/impl/gemini_service.go

```
@@ -2,28 +2,35 @@ package impl
+		geminiRedisCacheKey = "gemini_" + config.EnvGeminiModel() + "_content"
+	})
+
 	return &GeminiService{client, repository, cache}
 }
 
-// TODO: may need to move this to redis
-var geminiContent []*genai.Content
-
 func (srv *GeminiService) ResetSession() (string, error) {
-	geminiContent = nil
+	srv.cache.Delete(context.TODO(), geminiRedisCacheKey)
```

golangci-lint (1.64.8)

Error return value of `srv.cache.Delete` is not checked

(errcheck)